src/6-distributions/index.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2016-01-29
+id: 6-distributions
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "6-distributions", link: "./" }
+tags: dist, distributions
+title: IPFS distributions
+author: Richard Littauer
+collection: posts
+---
+
+[![](!img/screenshot.png)](http://dist.ipfs.io/)
+
+[dist.ipfs.io](http://dist.ipfs.io/) is the new distributions page for IPFS. This is the new one-stop-shop for finding and downloading all official binaries that IPFS produces.
+
```

Comment:
I had a site section, but removed it because I don't understand how the hashes work over time. See this comment: https://github.com/ipfs/blog/pull/23#discussion_r53617224